code/datums/status_effects/debuffs.dm

```diff
@@ -1299,3 +1301,70 @@
 	REMOVE_TRAIT(owner, TRAIT_PHYSICAL_HEALING_BLOCKED, STATUS_EFFECT_TRAIT)
 	if(ishuman(owner))
 		REMOVE_TRAIT(owner, TRAIT_SANITY_HEALING_BLOCKED, STATUS_EFFECT_TRAIT)
+
+// Tremor
+/datum/status_effect/stacking/lc_tremor
+	id = "lc_tremor"
+	alert_type = /atom/movable/screen/alert/status_effect/lc_tremor
+	max_stacks = 50
+	tick_interval = 10 SECONDS
+	consumed_on_threshold = FALSE
+	var/new_stack = TRUE
+
+/atom/movable/screen/alert/status_effect/lc_tremor
+	name = "Tremor"
+	desc = "You're unsteady on your feet, and move a bit slower."
+	icon = 'ModularTegustation/Teguicons/status_sprites.dmi'
+	icon_state = "tremor"
+
+//Slowdown on stack, prepares tremor burst
+/datum/status_effect/stacking/lc_tremor/on_apply()
+	. = ..()
+	owner.add_or_update_variable_movespeed_modifier(/datum/movespeed_modifier/tremor, multiplicative_slowdown = stacks * 0.4)
+
+/datum/status_effect/stacking/lc_tremor/on_remove()
+	owner.remove_movespeed_modifier(/datum/movespeed_modifier/tremor)
+	return ..()
+
+/datum/status_effect/stacking/lc_tremor/add_stacks(stacks)
+	. = ..()
+	owner.add_or_update_variable_movespeed_modifier(/datum/movespeed_modifier/tremor, multiplicative_slowdown = stacks * 0.4)
+
+/datum/status_effect/stacking/lc_tremor/can_have_status()
+	return (owner.stat != DEAD || !(owner.status_flags & GODMODE))
+
+// The Stack Decaying
+/datum/status_effect/stacking/lc_tremor/tick()
+	if(new_stack)
+		new_stack = FALSE
+	else
+		qdel(src)
+
+/datum/status_effect/stacking/lc_tremor/proc/TremorBurst()
+	new /obj/effect/temp_visual/weapon_stun/tremorburst(get_turf(owner))
+	playsound(owner, 'sound/effects/tremorburst.ogg', 50, FALSE)
+	if(ishuman(owner))
+		owner.Knockdown(stacks)
+		qdel(src)
+		return
+	owner.adjustBruteLoss(5 * stacks)
+	qdel(src)
+
+//Mob Proc
+/mob/living/proc/apply_lc_tremor(stacks, tremorburst)
+	var/datum/status_effect/stacking/lc_tremor/T = src.has_status_effect(/datum/status_effect/stacking/lc_tremor)
+	if(!T)
+		src.apply_status_effect(/datum/status_effect/stacking/lc_tremor, stacks)
+		new /obj/effect/temp_visual/damage_effect/tremor(get_turf(src))
+		return
+	else if(T.stacks >= tremorburst)
+		T.TremorBurst()
+	else
+		T.add_stacks(stacks)
+		new /obj/effect/temp_visual/damage_effect/tremor(get_turf(src))
+		T.new_stack = TRUE
```

Comment:
what IS this giant ass if else statement, why is there a return in there if there's no code following it
```suggestion
	if(!T)
		src.apply_status_effect(/datum/status_effect/stacking/lc_tremor, stacks)
		new /obj/effect/temp_visual/damage_effect/tremor(get_turf(src))
		return

	if(T.stacks < tremorburst)
		T.add_stacks(stacks)
		new /obj/effect/temp_visual/damage_effect/tremor(get_turf(src))
		T.new_stack = TRUE
		return
	T.TremorBurst()
```